src/components/features/sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { Location } from './components/cities';
+
+interface ModalWindow {
```

Comment:
catch you. Why you name if ModalWindow if you apply it to the Sidebar? it should be SidebarProps